sample/src/main/java/kin/sdk/core/sample/DisplayCallback.java

```diff
@@ -12,49 +11,32 @@
  */
 public abstract class DisplayCallback<T> implements ResultCallback<T> {
 
-    private WeakReference<View> progressBarReference;
-    private WeakReference<View> displayViewReference;
+    private View progressBar;
```

Comment:
not sure why you deleted WeakReference

Reply:
as we have cancellation now, weak reference is unnecessary safety measurements, I think that it's preferable to show that we trust our cancellation to detach the callbacks safely.